benefits/core/models.py

```diff
@@ -108,6 +110,43 @@ def by_id(id):
         logger.debug(f"Get {EligibilityVerifier.__name__} by id: {id}")
         return EligibilityVerifier.objects.get(id=id)
 
+    def get_verified_types(self, form=None, agency=None, oauth_claim=None):
```

Comment:
Since it can be tricky to know which combination of kwargs to use here, I propose we split this function into two:

* One for API verification w/`form` and `agency`
* Another for oauth verification w/`oauth_claim`